src/sentry/incidents/subscription_processor.py

```diff
@@ -336,78 +335,6 @@ def get_crash_rate_alert_aggregation_value(
 
     def get_crash_rate_alert_metrics_aggregation_value(
         self, subscription_update: QuerySubscriptionUpdate
-    ) -> float | None:
-        """
-        Handle both update formats.
-        Once all subscriptions have been updated to v2,
-        we can remove v1 and replace this function with current v2.
-        """
-        rows = subscription_update["values"]["data"]
-        if BaseCrashRateMetricsEntitySubscription.is_crash_rate_format_v2(rows):
-            version = "v2"
-            result = self._get_crash_rate_alert_metrics_aggregation_value_v2(subscription_update)
-        else:
-            version = "v1"
-            result = self._get_crash_rate_alert_metrics_aggregation_value_v1(subscription_update)
-
-        metrics.incr(
-            "incidents.alert_rules.get_crash_rate_alert_metrics_aggregation_value",
-            tags={"format": version},
-            sample_rate=1.0,
-        )
-        return result
-
-    def _get_crash_rate_alert_metrics_aggregation_value_v1(
```

Comment:
Looks like there's a test case from somewhere not being met: https://github.com/getsentry/sentry/actions/runs/11075656230/job/30777097641?pr=78275

Reply:
I don't have a way to store crash rate data locally, but everything I removed was explicitly labeled as such (including tests). I'll let the ingest team approve before I merge though since they have the most context.